tests/py/test_to_backend_api.py

```diff
@@ -19,8 +19,11 @@ def setUp(self):
                     "refit": False,
                     "debug": False,
                     "strict_types": False,
-                    "allow_gpu_fallback": True,
-                    "device_type": "gpu",
+                    "device": {
```

Comment:
@andi4191 This struct looks correct to you right?

Reply:
I think you are missing dla_core?